backend/api/overview_metrics.py

```
@@ -395,6 +395,21 @@ def _utc_midnight_ts(d):
     return int(datetime(d.year, d.month, d.day, tzinfo=dt_timezone.utc).timestamp())
 
 
+def _shift_months(d, months):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_shift_months`

(ANN202)